src/main/kotlin/racingcar/RacingGame.kt

```diff
@@ -10,20 +9,24 @@ class RacingGame(
     private val gameResult = mutableListOf<RaceResult>()
 
     constructor(
-        numberOfCars: Int,
+        carNames: List<String>,
         rounds: Int,
     ) : this(
-        Cars(List(numberOfCars) { Car() }),
+        Cars.fromNames(carNames),
         rounds,
     )
 
+    val currentResult: List<RaceResult>
+        get() = gameResult.toList()
```

Comment:
> This is a common code pattern among Java developers (including myself 😂)
To make it feel more Kotlin-like, consider using property instead of function.

I've changed like this 😊 Using properties seems much simpler and intuitive.